rts/lra/jaxrs/src/main/java/io/narayana/lra/filter/ServerLRAFilter.java

```diff
@@ -78,10 +83,13 @@
     private static final String TERMINAL_LRA_PROP = "terminateLRA";
     private static final String SUSPENDED_LRA_PROP = "suspendLRA";
     private static final String NEW_LRA_PROP = "newLRA";
+    private static final String ABORT_WITH_PROP = "abortWith";
 
     private static final Pattern START_END_QUOTES_PATTERN = Pattern.compile("^\"|\"$");
     private static final long DEFAULT_TIMEOUT_MILLIS = 0L;
-
+    private static final String JOIN = "join";
+    // i18nMessageCode corresponding to lraI18NLogger#warn_LRAStatusInDoubt
+    private static final int i18nMessageCode = 25145;
```

Comment:
What is the reason to explicitly provide the message code when it's printed by logger when logged?

Reply:
Take a look at where the constant gets used.
It provides a way for correlating the response with what's in the logs.

Reply:
I am not following your logic, all I've done is to include the log message id in the response to the client. I am not advocating anything complex here, we just need a story to tell the user.